admin/package.json

```diff
@@ -44,7 +44,7 @@
     "fast-csv": "^2.4.1",
     "feature-toggles": "^1.4.0",
     "fs-extra": "^7.0.1",
-    "govuk-frontend": "^2.13.0",
+    "govuk-frontend": "^3.0.0",
```

Comment:
isn't this and the yarn lock update in the other PR?

Reply:
@GuyHarwood yes. I merged the other branch in this one so that I could work with the latest toolkit version.